src/controllers/update/remove.is.entity.registered.owner.controller.ts

```diff
@@ -0,0 +1,21 @@
+import { NextFunction, Request, Response } from "express";
+import * as config from "../../config";
+import { logger } from "../../utils/logger";
+
+export const get = (req: Request, res: Response, _: NextFunction) => {
+  logger.debugRequest(req, `GET ${config.REMOVE_IS_ENTITY_REGISTERED_OWNER_PAGE}`);
```

Comment:
we should wrap the get and post function content in try. /catch blocks so we can show the error page if something goes wrong, i think the browser will just hang otherwise.

Reply:
Does this apply to sold land filter page as well?

Reply:
if it doesn't have the try catch then yes

Reply:
Updated sold land page with try catch. @mwestacott as you worked on this would you be able to take a look at this update just to make sure you are happy with what has been done here?

Reply:
👍 will do.